core/src/main/java/jenkins/telemetry/Telemetry.java

```diff
@@ -141,6 +141,22 @@ public static boolean isDisabled() {
         return jenkins == null || !jenkins.isUsageStatisticsCollected();
     }
 
+    /**
+     * Returns true iff we're in the time period during which this is supposed to collect data.
```

Comment:
better to avoid special symbols to keep Javadoc happy

Reply:
I don't understand this comment.

Reply:
```suggestion
     * Returns true if and only if we are in the time period during which this is supposed to collect data.
```